demos/select.html

```diff
@@ -176,40 +174,47 @@ <h2>Select Multiple - CSS Only</h2>
     </main>
     <script src="/assets/material-components-web.js"></script>
     <script>
-      (function() {
+      setTimeout(function() {
         var MDCSelect = mdc.select.MDCSelect;
-        var root = document.getElementById('js-select');
-        var currentlySelected = document.getElementById('currently-selected');
-        var select = MDCSelect.attachTo(root);
+        var heroSelect = document.getElementById('hero-js-select');
+        var heroSelectComponent = new mdc.select.MDCSelect(heroSelect);
+        heroSelectComponent.ripple.layout();
+      }, 300);
+    </script>
+    <script>
+      setTimeout(function() {
+        var MDCSelect = mdc.select.MDCSelect;
+        var root = document.getElementById('js-select-box');
+        var boxCurrentlySelected = document.getElementById('box-currently-selected');
+        var select = new MDCSelect(root);
+        select.ripple.layout();
+
         root.addEventListener('MDCSelect:change', function() {
           var item = select.selectedOptions[0];
           var index = select.selectedIndex;
-          currentlySelected.textContent = '"' + item.textContent + '" at index ' + index +
+          boxCurrentlySelected.textContent = '"' + item.textContent + '" at index ' + index +
            ' with value "' + select.value + '"';
```

Comment:
I know we don't lint our demo code, but could we possibly reindent this line while we're here?

Reply:
Sure can